dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -61,6 +62,17 @@ public void setup() throws Exception
         super.setup();
     }
 
+    
+    @Test
+    public void readBadlyDistributedFileTest() throws Exception
+    {
+        //this file only has data on two nodes
+        HPCCFile file = new HPCCFile("~test::alldatatypes", connString , hpccUser, hpccPass);
```

Comment:
is "test::alldatatypes" inherently skewed, or should we ensure skewed distribution?

Reply:
yes, it's inherently skewed (all data is on two nodes)